### PR TITLE
feat(content): Unfettered Hai, Teeneep, and Coalition now capitalize "Jump Drive"

### PR DESCRIPTION
**Pull request by ziproot**
----------------------
**Content (Artwork / Missions / Jobs)**

## Summary
As mentioned in #8536, both EjoThims and I preferred "Jump Drive" being capitalized. This PR is an alternate to #8536 where it is capitalized instead of lowercase. I kept all of the conditions, outfit names, etc. the same to preserve compatibility, but I updated anything that is in a text, conversation, etc.

~~**EDIT: After some discussion, I have made this PR a draft and opened discussion #8551, where we can discuss when to capitalize "Jump Drive" and when to leave it lowercase.**~~

**EDIT 2: I have reopened the PR. It only now applies to dialogue made by the Unfettered Hai, Teeneep, and the Coalition. Other instances of capitalization may come later, if this PR gets merged.**

## Save File
These save files can be used to play through the new mission content: 
[Endless-Sky any%~Coalition Expeditions Past Start.txt](https://github.com/endless-sky/endless-sky/files/11069864/Endless-Sky.any.Coalition.Expeditions.Past.Start.txt)
[Endless-Sky any%~Coalition Jump Drive Job.txt](https://github.com/endless-sky/endless-sky/files/11069865/Endless-Sky.any.Coalition.Jump.Drive.Job.txt)
[Endless-Sky any%~Coalition Jump Drive Mission.txt](https://github.com/endless-sky/endless-sky/files/11069866/Endless-Sky.any.Coalition.Jump.Drive.Mission.txt)
[Endless-Sky any%~Coalition Kimek Gift.txt](https://github.com/endless-sky/endless-sky/files/11069867/Endless-Sky.any.Coalition.Kimek.Gift.txt)
[Endless-Sky any%~Coalition Longcow Antibiotics.txt](https://github.com/endless-sky/endless-sky/files/11069868/Endless-Sky.any.Coalition.Longcow.Antibiotics.txt)
[Endless-Sky any%~Coalition Lost Jump Drive.txt](https://github.com/endless-sky/endless-sky/files/11069869/Endless-Sky.any.Coalition.Lost.Jump.Drive.txt)
[Endless-Sky any%~Coalition Saryd University Lecture.txt](https://github.com/endless-sky/endless-sky/files/11069870/Endless-Sky.any.Coalition.Saryd.University.Lecture.txt)
[Endless-Sky any%~First Contact: Coalition.txt](https://github.com/endless-sky/endless-sky/files/11069871/Endless-Sky.any.First.Contact.Coalition.txt)
[Endless-Sky any%~First Contact: Unfettered.txt](https://github.com/endless-sky/endless-sky/files/11069872/Endless-Sky.any.First.Contact.Unfettered.txt)
[Endless-Sky any%~Hai Reveal Journey to Freedom.txt](https://github.com/endless-sky/endless-sky/files/11069873/Endless-Sky.any.Hai.Reveal.Journey.to.Freedom.txt)
[Endless-Sky any%~Hai Reveal Moving Together.txt](https://github.com/endless-sky/endless-sky/files/11069874/Endless-Sky.any.Hai.Reveal.Moving.Together.txt)
[Endless-Sky any%~Unfettered Jump Drives.txt](https://github.com/endless-sky/endless-sky/files/11069875/Endless-Sky.any.Unfettered.Jump.Drives.txt)
[Endless-Sky any%~Wanderers Jump Drive Source.txt](https://github.com/endless-sky/endless-sky/files/11069876/Endless-Sky.any.Wanderers.Jump.Drive.Source.txt)

## Artwork Checklist
~~- [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
~~- [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~~
~~- [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

### CODE REVIEW

File: data/remnant/remnant jobs.txt [hunk 1]
```diff
@@ -100,11 +100,11 @@ mission "Remnant: Broken Jump Drive job"
 		dialog phrase "broken jump drive on visit"
 	on complete
 		outfit "Jump Drive (Broken)" -1
-		"remnant: broken jump drive count" ++
+		"remnant: broken Jump Drive count" ++
```
Comment (Zitchas) [inline data/remnant/remnant jobs.txt:103]:
```suggestion
		"remnant: broken jump drive count" ++
```

This is a condition, not proper text, so it should remain lower case.

**Review by Zitchas — CHANGES_REQUESTED** [review]
Looks good, overall. I like the way it looks with Jump Drive capitalized rather than lowercase. Lowercase just looks like it's some random junk thing of no consequence, not an important, rare, and unique outfit.

One instance of a condition getting capitalized, though.

File: data/remnant/remnant jobs.txt (hunk 1 above)
Comment (ziproot) [reply data/remnant/remnant jobs.txt:103]: Oops. I thought I got all those.

**Comment by ziproot** [comment]
There are two comments that I capitalized that include the word "Jump Drive," both of which are in Hai Reveal. I am unsure as to whether or not they should be capitalized. If people agree, I will  revert those changes:

	
	# Due to the extremely limited number of Jump Drives, the Devil-Run
	# pirate gang only has the largest of ships. They used Jump Drives in...
---
	# IMPORTANT: For story reasons, all non-Unfettered ships must have "save" to ensure
	# the Unfettered get their Jump Drives back.

**Comment by tibetiroka** [comment]
See https://github.com/endless-sky/endless-sky/pull/8536#issuecomment-1467355232 for why I requested Derpy's review.

**Review by Zitchas — APPROVED** [review]
I like it. Really makes Jump Drives stand out like the supposedly special things they are.

**Review by EjoThims — APPROVED** [review]
This is my preference between the two as well, but either way I think the most important part is consistency in the game presentation.

**Comment by ziproot** [comment]
From what I read, Amazinite prefers "Jump Drive" is lowercase because multiple factions have them, making it a "generic" item. However, unlike most other items that would qualify, these are strictly controlled by select factions, so they fall in a weird grey area between "generic" and "specific to one faction."  I believe that this therefore deserves reconsideration, and a metric should be given for these "grey area" items, which at the moment is just the Jump Drive.

**Comment by EjoThims** [comment]
Not really appropriate to just copy/paste wholesale, but relevant comment on the other PR.

https://github.com/endless-sky/endless-sky/pull/8536#issuecomment-1471420563

**Comment by ziproot** [comment]
I am willing to have some of the factions refer to Jump Drives in uppercase and others refer to them in lowercase. I am currently going to turn this into a draft PR until we can decide about that.

**Comment by ziproot** [comment]
I have opened a discussion [here](https://github.com/endless-sky/endless-sky/discussions/8551).

**Comment by ziproot** [comment]
If anything is mentioned verbally during Amazinite's stream that is relevant to the capitalization of "Jump Drive," please post it as a comment in the linked discussion. Thank you.

**Comment by ziproot** [comment]
As mentioned in the discussion, when I get the time I will remove all of the changes to capitalization that aren't in Unfettered Hai and Coalition and reopen this PR. Those other changes can be added in future PRs after further discussion.

**Comment by ziproot** [comment]
Update: I have removed capitalization from almost everyone for "phase 1" of Jump Drive Capitalization. Only the Unfettered, Coalition, and Teeneep, when speaking, capitalize it. I included Teeneep as they are mentioned to be true Hai by the Unfettered and so they may capitalize Jump Drive. If people feel like this should not be the case I am willing to also revert those changes.

**Comment by ziproot** [comment]
I updated the description to fit the template. I do not have save files yet. I will have to actually play Endless Sky (*gasp*) in order to create the save files. I can expedite the process of getting money and ships through Omnis and save file editing but I will still need to speedrun the FW campaign. Expect a save file in the next few days.

**Comment by ziproot** [comment]
Apparently landing on one Kor Efreti planet, then leaving, is enough for you to know the backstory of the Korath, that the Hai exist, and that there is a faction called the Unfettered who wants jump drives, but only for the duration of the lecture.

**Comment by ziproot** [comment]
I have updated the PR with the save files. Some of them require you to visit spaceport to activate the mission, others require you to go to the right planet, etc, but it should take at most five minutes to go from a given save file to a mission that was updated by this PR.

**Comment by ziproot** [comment]
The recently merged PR #8536 has created zero conflicts with this PR due to the reduced function. It modified logs to be lowercase and this only touches dialogue.

**Review by Hurleveur — APPROVED** [review]
I personaly agree that we should capitalize it, mostly because it makes the text easier to read and shows it is an actual outfit, not just a commodity or something of no importance.

**Comment by Hurleveur** [comment]
Btw I dont think you need that many save files, especially not for simple wording changes, but I could be wrong.
Not that its a bad thing to have them - just feels like a lot of unnessecary work?

**Review by EjoThims — APPROVED** [review]
Just refreshing this with the changes :)

**Comment by Amazinite** [comment]
If we're changing the capitalization of anything, it should be all or nothing, not a matter of some files capitalize and others don't, or capitalized in dialog but not capitalized in narration.

**Comment by ziproot** [comment]
Well, there goes my attempt at a compromise. Can you explain why you think that everything should be the same? Not all species see the "Jump Drive" the same way. Some might see it as a generic thing while others, like the Unfettered Hai, Coalition, and possibly Teeneep, see it as more than that.

**Comment by roadrunner56** [comment]
I'd rather our rules for capitalizing something be all or nothing. I think that capitalizing 'jump drive' based on rather subjective aspects of a species' culture will just lead to more messy arguments down the line. Also, I think it just looks kind of sloppy to be inconsistent with capitalization.

**Comment by mOctave** [comment]
I still think it could work well when capitalized by select groups in dialog, but I can see how it would look sloppy to people who didn't know the reasons, and so it might be best avoided after all for the reasons presented here.

**Comment by ziproot** [comment]
It will take me some time to revert my commit reducing the capitalization to a few cases without undoing other commits. I am willing to do so if people agree this is the better option.

**Comment by roadrunner56** [comment]
I personally think it shouldn't be capitalized, but then again I'm not a dev.

**Comment by Saugia** [comment]
I have to agree with it not being capitalized, especially to only specific factions.

**Comment by ziproot** [comment]
In that case, I will close this PR. If Zitchas, Amazinite, and Saugia can agree on a standard for when to capitalize "Jump Drive" that requires changes, I am willing to commit those changes.

**Pull request closed without merging**